[improve][doc] relocate "Get Started" (pulsar-admin) to admin API and correct inaccurate descriptions

Anonymitaet opened the pull request:
This PR fixes https://github.com/apache/pulsar/issues/20024

I’ve summarized all issues with the corresponding solutions below. 

Issues | Solutions
-- | --
[Get Started of pulsar-admin ](https://pulsar.apache.org/docs/next/get-started-pulsar-admin/)is shown in [Tutorials](https://pulsar.apache.org/docs/next/how-to-landing/). | Relocate it to [Admin API - Get Started](https://pulsar.apache.org/docs/next/admin-api-get-started/) to improve the ease of use.
The wordings "Java admin API" in [Admin API > Features > Functions](https://pulsar.apache.org/docs/next/admin-api-functions/) and [Admin API > Features > Schemas](https://pulsar.apache.org/docs/next/admin-api-schemas/) are inaccurate. | Correct it to "Java".

- [x] `doc` 

@labuladong @asafm @jak78 thanks for your technical guidance!

Ping @D-2-Ed to review. Thank you!

Anonymitaet commented:
Preview looks good:
![image](https://user-images.githubusercontent.com/50226895/230266036-8e3f4dc3-2862-407c-9c96-072de623fc29.png)

Anonymitaet commented:
@asafm @jak78 Could you please review this PR from a technical perspective?

jak78 commented:
Thanks @Anonymitaet
Can you check the links? It seems that some are broken. For example: 
![Screenshot 2023-04-06 at 13 02 46](https://user-images.githubusercontent.com/351773/230358832-469d7b3c-8dc5-4757-8b26-9867b3d5b402.png)
'Install and start Pulsar standalone' looks broken.
This one also:
![Screenshot 2023-04-06 at 13 09 47](https://user-images.githubusercontent.com/351773/230360273-9fc90e10-62a9-42fc-9156-8f15c629ab21.png)

jak78 commented:
![Screenshot 2023-04-06 at 13 14 01](https://user-images.githubusercontent.com/351773/230361083-2df43fb6-0859-4ef3-a880-70c30f6eaa97.png)
I could not find any Pulsar Java Admin API example in the provided repo.
I suggest the following text instead:

> Prepare a Java project. You can add the following dependency to your POM:
```xml
    <dependency>
        <groupId>org.apache.pulsar</groupId>
        <artifactId>pulsar-client-admin</artifactId>
        <version>2.11.0</version>
    </dependency>
```

jak78 commented:
I have some fixes to the Java code samples: https://github.com/jak78/pulsar-site/commit/16187c78765b5b274d46b2e62b77b75f0e54c3b3

Anonymitaet commented:
Hi @jak78 thanks for your guidance!

- For your 1st suggestion:

  Solution: I've changed them to plain text as a temporary workaround.

  Reason: I've made some tests and found that `.md`-type URLs can be processed correctly in plain texts rather than in "Tabs". I guess it's a Docusarus limitation.

<img width="985" alt="image" src="https://user-images.githubusercontent.com/50226895/230368356-dd3169a6-2f35-413a-9e0c-2d7fad6516bf.png">

<img width="922" alt="image" src="https://user-images.githubusercontent.com/50226895/230371836-77744be5-2744-4bbd-85b8-72a21a6b9c3e.png">

- For your 3rd suggestion: I've incorporated your comment, PTAL.

jak78 commented:
LGTM, thanks

Anonymitaet commented:
Hi @jak78 

For your 2nd comment: I've double-checked this with @hangc0276, he did a test on it, and it works! 
So I've updated the doc as below.
<img width="695" alt="image" src="https://user-images.githubusercontent.com/50226895/230591187-37ad60d7-5184-4197-8163-018d1929292d.png">

Anonymitaet commented:
@momo-jun @Huanli-Meng could you please help review? Thank you!

Huanli-Meng commented on docs/admin-get-started.md:
````diff
@@ -9,102 +9,245 @@ import Tabs from '@theme/Tabs';
+    brokerServiceUrl=pulsar://localhost:6650/
+    ```
+
+2. Create a persistent topic named test-topic-1 with 6 partitions.
````
```suggestion
2. Create a persistent topic named _test-topic-1_ with 6 partitions.
```

Anonymitaet replied:
Oops! Updated.

Huanli-Meng commented on docs/admin-get-started.md:
```diff
@@ -9,102 +9,245 @@ import Tabs from '@theme/Tabs';
+
+    There is no output. You can check the number of partitions in Step 5.
+
+4. Produce some messages to the partitioned topic test-topic-1.
```
```suggestion
4. Produce some messages to the partitioned topic _test-topic-1_.
```

Huanli-Meng commented on docs/admin-get-started.md:
```diff
@@ -9,102 +9,245 @@ import Tabs from '@theme/Tabs';
-    }
-    producer.close();
-    client.close();
+    Below is a part of the output. For detailed explanations of topic stats, see Pulsar statistics.
```
I would suggest that we can add a link to Pulsar statistics.

Anonymitaet replied:
Currently, we can not add a link here. I've explained the reason in the comment above https://github.com/apache/pulsar-site/pull/504#issuecomment-1498946786

Huanli-Meng commented on docs/admin-get-started.md:
````diff
@@ -9,102 +9,245 @@ import Tabs from '@theme/Tabs';
     ```
 
-2. Create a partitioned topic _test-topic-1_ with 4 partitions.
+    **Output**
````
I would suggest we use the note to tell users there is no output here instead of using **Output** here. same comments for the next step.

Anonymitaet replied:
We do not need to add this kind of "tip" since it's basic knowledge for devs :-D 
Same for Linux. No result is the best result (the operation is successful)

Huanli-Meng reviewed (approved):
Leave some minor comments.
